sdk/.env.example

```diff
@@ -1,8 +1,26 @@
+# This file contains environment variables that are used and shared by our SDK examples.
+# Make sure to change mnemonics/seeds prior to running any examples in order to prevent
+# any interference by other users running those examples.
+
+# Mnemonics (Don't ever use them to manage real funds!)
 NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1="endorse answer radar about source reunion marriage tag sausage weekend frost daring base attack because joke dream slender leisure group reason prepare broken river"
 NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_2="width scatter jaguar sponsor erosion enable cave since ancient first garden royal luggage exchange ritual exotic play wall clinic ride autumn divert spin exchange"
-NON_SECURE_USE_OF_DEVELOPMENT_SEED_1=0x256a818b2aac458941f7274985a410e57fb750f3a3a67969ece5bd9ae7eef5b2
-NON_SECURE_USE_OF_DEVELOPMENT_SEED_2=0x256a818b2aac458941f7274985a410e57fb750f3a3a67969ece5bd9ae7eef5b3
-STRONGHOLD_PASSWORD="some_hopefully_secure_password"
+# Seeds (Don't ever use them to manage real funds!)
+NON_SECURE_USE_OF_DEVELOPMENT_SEED_1="0x256a818b2aac458941f7274985a410e57fb750f3a3a67969ece5bd9ae7eef5b2"
```

Comment:
lets get rid of seed completely?

Reply:
yeah

Reply:
created an issue: https://github.com/iotaledger/iota-sdk/issues/458 
Okay with this @kwek20 ?